src/DGmethods/test/compressible_Navier_Stokes/soundings/create_sounding.m

```diff
@@ -0,0 +1,134 @@
+clear all
```

Comment:
I think that we should convert this into Julia and skip adding the `.dat` files into the repo.